src/eos-kolibri-export

```diff
@@ -38,9 +40,9 @@ def system_server_proxy():
         bus,
         Gio.DBusProxyFlags.NONE,
         None,  # interface info
-        'org.learningequality.Kolibri.Daemon',
-        '/org/learningequality/Kolibri/Daemon/Main',
-        'org.learningequality.Kolibri.Daemon',
+        KOLIBRI_FLATPAK_ID + '.Daemon',
+        KOLIBRI_FLATPAK_ID.replace('.', '/') + '/Daemon/Main',
+        KOLIBRI_FLATPAK_ID + '.Daemon',
```

Comment:
I think it would be better if you added `KOLIBRI_DAEMON_SERVICE` to `config.py` and used that since this is clearly about the DBus interface. It would also be nice to do the conversion from bus/interface name to object name once there.

Reply:
Or maybe it would be better to move this to `eos-kolibri-export.in` and substitute the values at build time as both `eos-kolibri-export` and `endless-key-export` like the other scripts.

Reply:
> I think it would be better if you added KOLIBRI_DAEMON_SERVICE to config.py and used that since this is clearly about the DBus interface. It would also be nice to do the conversion from bus/interface name to object name once there.

AH!  Here should be FLATPAK_ID + '.Daemon',  and so on ...
But, having bus, object and interface names sounds good!

> Or maybe it would be better to move this to `eos-kolibri-export.in` and substitute the values at build time as both `eos-kolibri-export` and `endless-key-export` like the other scripts.

`eos-kolibri-{import, export}.in` => `eos-kolibri-{import, export}` and `endless-key-{import, export}` is an interesting idea!  However, we can only have one `config.py`.  So, keeping `eos-kolibri-import`, `eos-kolibri-export` and `eos_kolibri/eos_kolibri_migrate/main.py` might be better.